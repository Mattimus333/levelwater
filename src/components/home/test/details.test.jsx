import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import Details from '../details.jsx';


describe('details', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Details />, div);
  });
  it('renders a snapshot', () => {
    const tree = shallow(<Details />);
    expect(tree).toMatchSnapshot();
  });
});
