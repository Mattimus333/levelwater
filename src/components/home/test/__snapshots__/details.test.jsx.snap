exports[`details renders a snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    id="details">
    <div
        className="ui grid">
        <div
            className="column sixteen wide">
            <h3>
                More Details
            </h3>
        </div>
        <div
            className="column eight wide">
            <p>
                More
            </p>
            <ul>
                <li>
                    -Lorem ipsum dolor sit amet,
                </li>
                <li>
                    -consectetur adipisicing elit.
                </li>
                <li>
                    -Tempora dolores, dolore quisquam ea minus
                </li>
                <li>
                    -veniam labore minima cupiditate sunt
                </li>
            </ul>
        </div>
        <div
            className="column eight wide">
            <p>
                Details
            </p>
            <ul>
                <li>
                    -Lorem ipsum dolor sit amet,
                </li>
                <li>
                    -consectetur adipisicing elit.
                </li>
                <li>
                    -Tempora dolores, dolore quisquam ea minus
                </li>
                <li>
                    -veniam labore minima cupiditate sunt
                </li>
            </ul>
        </div>
    </div>
</div>,
  "nodes": Array [
    <div
      id="details">
      <div
            className="ui grid">
            <div
                  className="column sixteen wide">
                  <h3>
                        More Details
                  </h3>
            </div>
            <div
                  className="column eight wide">
                  <p>
                        More
                  </p>
                  <ul>
                        <li>
                              -Lorem ipsum dolor sit amet,
                        </li>
                        <li>
                              -consectetur adipisicing elit.
                        </li>
                        <li>
                              -Tempora dolores, dolore quisquam ea minus
                        </li>
                        <li>
                              -veniam labore minima cupiditate sunt
                        </li>
                  </ul>
            </div>
            <div
                  className="column eight wide">
                  <p>
                        Details
                  </p>
                  <ul>
                        <li>
                              -Lorem ipsum dolor sit amet,
                        </li>
                        <li>
                              -consectetur adipisicing elit.
                        </li>
                        <li>
                              -Tempora dolores, dolore quisquam ea minus
                        </li>
                        <li>
                              -veniam labore minima cupiditate sunt
                        </li>
                  </ul>
            </div>
      </div>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Details />,
      "_debugID": 20,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          id="details">
          <div
                    className="ui grid">
                    <div
                              className="column sixteen wide">
                              <h3>
                                        More Details
                              </h3>
                    </div>
                    <div
                              className="column eight wide">
                              <p>
                                        More
                              </p>
                              <ul>
                                        <li>
                                                  -Lorem ipsum dolor sit amet,
                                        </li>
                                        <li>
                                                  -consectetur adipisicing elit.
                                        </li>
                                        <li>
                                                  -Tempora dolores, dolore quisquam ea minus
                                        </li>
                                        <li>
                                                  -veniam labore minima cupiditate sunt
                                        </li>
                              </ul>
                    </div>
                    <div
                              className="column eight wide">
                              <p>
                                        Details
                              </p>
                              <ul>
                                        <li>
                                                  -Lorem ipsum dolor sit amet,
                                        </li>
                                        <li>
                                                  -consectetur adipisicing elit.
                                        </li>
                                        <li>
                                                  -Tempora dolores, dolore quisquam ea minus
                                        </li>
                                        <li>
                                                  -veniam labore minima cupiditate sunt
                                        </li>
                              </ul>
                    </div>
          </div>
</div>,
        "_debugID": 21,
        "_renderedOutput": <div
          id="details">
          <div
                    className="ui grid">
                    <div
                              className="column sixteen wide">
                              <h3>
                                        More Details
                              </h3>
                    </div>
                    <div
                              className="column eight wide">
                              <p>
                                        More
                              </p>
                              <ul>
                                        <li>
                                                  -Lorem ipsum dolor sit amet,
                                        </li>
                                        <li>
                                                  -consectetur adipisicing elit.
                                        </li>
                                        <li>
                                                  -Tempora dolores, dolore quisquam ea minus
                                        </li>
                                        <li>
                                                  -veniam labore minima cupiditate sunt
                                        </li>
                              </ul>
                    </div>
                    <div
                              className="column eight wide">
                              <p>
                                        Details
                              </p>
                              <ul>
                                        <li>
                                                  -Lorem ipsum dolor sit amet,
                                        </li>
                                        <li>
                                                  -consectetur adipisicing elit.
                                        </li>
                                        <li>
                                                  -Tempora dolores, dolore quisquam ea minus
                                        </li>
                                        <li>
                                                  -veniam labore minima cupiditate sunt
                                        </li>
                              </ul>
                    </div>
          </div>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Details />,
}
`;
